include/netlink/route/link/bridge_info.h

```diff
@@ -6,6 +6,7 @@
 #ifndef NETLINK_LINK_BRIDGE_INFO_H_
 #define NETLINK_LINK_BRIDGE_INFO_H_
 
+#include <stdbool.h>
```

Comment:
`<stdbool.h>` is a C99 header. So far we don't use that in public headers.

However, we already use `<stdint.h>` (also C99), so we can assume that all users use C99 and will also have <stdbool.h>.

One concern is that `<stdbool.h>` redefines `bool`/`true`/`false`. So if a user had their own such identifiers, this change will break them (well, at least, if they use this particular header -- which probably means that nobody is going to be affected). However, this also creates a precedence, so soon we might use `<stdbool.h>` in more prominent headers, where it might break some users.

Unclear what to do about that. Maybe just use an `int`?

Reply:
Personally I think that using an int over a bool for new code would probably be a step backwards. However, since your suggestion lines up nicely with your other suggestion, I will change it to use an int.